### PR TITLE
fix output var formatting for env

### PR DESCRIPTION
**Pull request by ddollar**

### CODE REVIEW

**Comment by ddollar** [comment]
[output var formatting for dev env](https://trello.com/c/B3sOE3Vf/1322-output-var-formatting-for-dev-env)

File: provider/aws/dist/rack.json [hunk 1]
```diff
@@ -156,7 +156,7 @@
         { "Ref": "ExistingVpc" }
       ] }
     },
-    "VPCCIDR": {
+    "Vpccidr": {
```
Comment (MiguelMoll) [inline provider/aws/dist/rack.json]: All caps was causing the weird underscores?

**Comment by MiguelMoll** [comment]
Anything to expect with removing the registry?

File: provider/aws/dist/rack.json (hunk 1 above)
Comment (ddollar) [inline provider/aws/dist/rack.json]: Yep

**Pull request closed without merging**